terraform/alb.tf

```diff
@@ -0,0 +1,79 @@
+resource "aws_security_group" "lb" {
+  name   = "eg-lb"
+  vpc_id = "${aws_vpc.event-gateway.id}"
+
+  ingress {
+    protocol    = "tcp"
+    from_port   = 80
+    to_port     = 80
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+}
+
+# Config load balancer
+resource "aws_lb" "config" {
```

Comment:
This module should probably simplify the setup here https://registry.terraform.io/modules/terraform-aws-modules/alb/aws/3.4.0?tab=resources

Reply:
The problem with this module is that it doesn't seem to allow to control target group health checks. We need to overwrite the default "/" path with "/v1/status". What's more, the events API server doesn't expose any health check endpoint returning 200 code (the root returns 202).

Reply:
Good point but looks like it should be possible.

![screenshot 2018-07-06 09 10 09](https://user-images.githubusercontent.com/455261/42364700-6842b43e-80fc-11e8-9803-4a92c75515b2.png)

Reply:
yes, I've missed the default section. Works perfectly with the alb module.